### PR TITLE
Update Localizable.strings for german

### PR DESCRIPTION
**Pull request by hirbod**

### CODE REVIEW

File: Sources/ZLImageEditor.bundle/de.lproj/Localizable.strings [hunk 1]
```diff
@@ -1,9 +1,9 @@
-"cancel" = "Stornieren";
+"cancel" = "Abbrechen";
```
Comment (longitachi) [inline Sources/ZLImageEditor.bundle/de.lproj/Localizable.strings:1]: Thanks for your pr. But I want to mean cancel, not abort. I see the text in WeChat is also 'Stornieren'.
image

Comment (hirbod) [reply Sources/ZLImageEditor.bundle/de.lproj/Localizable.strings:1]: Nobody in Germany uses "Stornieren". And WeChats translation is totally off anyway. As a native speaker, I can tell you that "Abbrechen" is the correct term here.

Comment (hirbod) [reply Sources/ZLImageEditor.bundle/de.lproj/Localizable.strings:1]: We use "Stornieren" only when we have like a running Payment or Shipping and wan't to cancel this out, but not in this context for a button.

Comment (longitachi) [reply Sources/ZLImageEditor.bundle/de.lproj/Localizable.strings:1]: OK~

**Review by longitachi — APPROVED** [review]

**Pull request merged**